crates/ruff/tests/lint.rs

```diff
@@ -2628,6 +2628,47 @@ class A(Generic[T]):
     );
 }
 
+#[test]
+fn walrus_before_py38() {
+    // ok
+    assert_cmd_snapshot!(Command::new(get_cargo_bin(BIN_NAME))
+        .args(STDIN_BASE_OPTIONS)
+        .args(["--stdin-filename", "test.py"])
+        .arg("--target-version=py38")
+        .arg("-")
+        .pass_stdin(r#"if x := 1: pass"#),
```

Comment:
We can just use `(x := 1)` to avoid unrelated diagnostics in the snapshot. Same for other test cases.